### ISSUE TITLE
About the test file

### ISSUE THREAD

**Issue by jqc8438** [yassouali/pytorch_segmentation#94]
Thanks for your great code!
I am a beginner for segmentation.I trained the model(PSPnet in Pascal VOC) on my computer.And I want to know the test mIoU,but i fail to find  the text.py.
anything i forgot?

**Comment by yassouali** [comment]
Hi @jqc8438 
Maybe ` inference.py ` is the file you are looking for.

**Comment by jqc8438** [comment]
Really glad to receive your response.
Actually, i have used the `inference.py`. I can use it to segment the image.However, I did not find that it has the function of calculating mIoU on the VOC test set .It just get the segmented picture and put it in the `outputs` folder. 
And i  just want to evaluate the effect (mIoU) of the model on the test set(not only on Val set). 
Did I miss any details?
Thank you so much for your help！

**Comment by yassouali** [comment]
To get the mIoU on the test set, you'll need to save the results as png images and submit the results using the pascal voc evaluation server: http://host.robots.ox.ac.uk:8080/

**Comment by jqc8438** [comment]
get it!
thank you so much!
